source/time_tables/departure_boards.h

```diff
@@ -47,6 +47,22 @@ void departure_board(PbCreator& pb_creator, const std::string &filter,
                      int count, int start_page,
                      const type::RTLevel rt_level,
                      const size_t items_per_route_point);
+
+time_duration to_navitia (boost::posix_time::time_duration dur);
```

Comment:
do you want to expose these functions? If not, place them in the .cpp and declare them as `static` functions
http://stackoverflow.com/a/25724838/1614576